templates/olm-catalog/kubevirt-hyperconverged/kubevirt-hyperconverged.package.yaml.in

```diff
@@ -1,4 +1,7 @@
 packageName: kubevirt-hyperconverged
 channels:
```

Comment:
This seems so strange to me...the idea that you can't have alpha and stable channels. Mind blowing.

Reply:
The issue is the quay app registry.  It doesn't support anything except incremental numeric releases e.g. 2.0.0, 2.0.1, 3.0.0.  This method would only work if you had a registry per release.